waku/v2/node/wakunode2.nim

```diff
@@ -838,6 +848,20 @@ when isMainModule:
     # Connect to configured static nodes
     if conf.staticnodes.len > 0:
       waitFor connectToNodes(node, conf.staticnodes)
+    
+    # Connect to discovered nodes
+    if conf.dnsDiscovery and conf.dnsDiscoveryUrl != "":
+      # @ TODO: this is merely POC integration with an empty resolver
```

Comment:
Maybe debug log that this is using empty resolver

Reply:
Done in ed2fdb2.